### PR TITLE
Fix docker image oops in hostpath-installation.md

### PR DESCRIPTION
**Pull request by matthewadams**

### CODE REVIEW

File: docs/main/user-guides/local-storage-user-guide/local-pv-hostpath/hostpath-installation.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Set up the directory on the nodes where Local PV Hostpaths will be created. This
 :::note
 **air-gapped environment**
 If you are running your Kubernetes cluster in an air-gapped environment, make sure the following container images are available in your local repository.
-- openebs/localpv-provisioner
+- openebs/provisioner-localpv
```
Comment (avishnu) [inline docs/main/user-guides/local-storage-user-guide/local-pv-hostpath/hostpath-installation.md:35]: Is this correct @Abhinandan-Purkait ?

Comment (Abhinandan-Purkait) [reply docs/main/user-guides/local-storage-user-guide/local-pv-hostpath/hostpath-installation.md:35]: Yes, the image name is provisioner-localpv. 

Ref: https://hub.docker.com/r/openebs/provisioner-localpv/tags

Comment (avishnu) [reply docs/main/user-guides/local-storage-user-guide/local-pv-hostpath/hostpath-installation.md:35]: Ok 👍

**Comment by avishnu** [comment]
@matthewadams ,
Thanks for the PR.

Can you amend the commit message to match the linter rules, like say
docs: fix docker image oops in hostpath-installation.md

And you need to dco sign your commit as well.

**Comment by matthewadams** [comment]
@avishnu ✅

**Comment by avishnu** [comment]
> @avishnu ✅

You will have to rebase as there are conflicts now.

**Comment by matthewadams** [comment]
Dude, it's a typo fix.  This is too much trouble.  Have a committer make the trivial change.  Closing PR.

**Comment by avishnu** [comment]
@matthewadams Unfortunately the conflicts are preventing the PR from getting merged. We'll make the change and credit to you. Thanks for your efforts !!

**Comment by orville-wright** [comment]
> @matthewadams , Thanks for the PR.
> 
> Can you amend the commit message to match the linter rules, like say docs: fix docker image oops in hostpath-installation.md
> 
> And you need to dco sign your commit as well.

@avishnu - I've discovered... when a person creates a PR directly via the GitHub WebApp (i.e. not using shell or Gitbash or linux shell from a full local repo branch copy etc)... the GitHub WebApp makes it almost impossible to change/modify the "commit message" once the initial PR commit is done. 

- I've tried, and you cannot change commit comments from the GitHub WebApp (I haven't found any way to do it).
- Changing the Commit comments are only possible via ``` cmd line ```, from a local repo branch via ``` commit --amend ``` or via an app like ``` VSCode ``` against a local branch repo (note: it might work directly against the remote copy... I've never tried).

So... trivial updates to webpages and docs updates; become overly burdensome if contributors are asked/required to constantly ``` change commit comments ``` and rebase the PR branch... just to help the LINTER Rules pass.

I'm not sure if @matthewadams created his PR this way?
I sympathize with @matthewadams here, as I've been dealing with this headache for the past few months and I can't find a smooth & painless way to resolve this issue. - Sorry @matthewadams ... I share your pain here.

I think we need to find a less burdensome process for our users who take the time and effort to edit, update, modify and enhance our webpage and docs content.  As you know, this is the most valuable feedback and community contributions that we can get, so we need to help them with a more painless process.

Just want to apologize once again @matthewadams - I share your pain here

**Pull request closed without merging**